frontend/src/components/inbox/pending-handover/PendingHandoverSearch.vue

```diff
@@ -0,0 +1,336 @@
+}
+
+function validateSearchInput() {
+  //Startdatum fehlt
+  if (
+    query.value?.decisionDateEnd &&
+    !query.value?.decisionDate &&
+    !validationStore.getByField("decisionDate")
+  ) {
+    validationStore.add("Startdatum fehlt", "decisionDate")
+  } else if (
+    !query.value.decisionDateEnd &&
+    validationStore.getByMessage("Startdatum fehlt").length === 1
+  ) {
+    validationStore.remove("decisionDate")
+  }
+
+  //Enddatum darf nicht vor Startdatum liegen
+  if (
+    query.value?.decisionDateEnd &&
+    query.value?.decisionDate &&
+    new Date(query.value.decisionDate) > new Date(query.value.decisionDateEnd)
+  ) {
+    if (!validationStore.getByField("decisionDateEnd")) {
+      validationStore.add(
+        "Enddatum darf nicht vor Startdatum liegen",
+        "decisionDateEnd",
+      )
+    }
+  } else if (
+    validationStore.getByMessage("Enddatum darf nicht vor Startdatum liegen")
+      .length === 1
+  ) {
+    validationStore.remove("decisionDateEnd")
+  }
+}
+
+function handleLocalInputError(error: ValidationError | undefined, id: string) {
+  if (error) {
+    validationStore.add(
+      error.message,
+      error.instance as DocumentUnitSearchParameter,
+    )
+  } else validationStore.remove(id as DocumentUnitSearchParameter)
+  validateSearchInput()
+}
+
+/**
+ * Checks if the current search query is identical to the previous query.
+ * @returns {boolean} - `true` if the current query matches the previous query, otherwise `false`.
+ */
+function isIdenticalSearch(): boolean {
+  const previousQuery = getQueryFromRoute()
+  const newQuery = query.value
+  return JSON.stringify(previousQuery) === JSON.stringify(newQuery)
+}
+
+function handleSearchButtonClicked() {
+  validateSearchInput()
+
+  if (isSearchInputInvalid()) {
+    return
+  }
+
+  if (isIdenticalSearch()) {
+    handleSearch()
+  }
+  pushQueryToRoute(query.value)
+}
+
+function handleSearch() {
+  if (!isEmptySearch.value) {
+    emit("search", getQueryFromRoute())
+  } else {
+    resetSearch()
+  }
+}
+
+watch(
+  route,
+  () => {
+    handleSearch()
+  },
+  { deep: true, immediate: true },
+)
+</script>
+
+<script lang="ts">
+export type DocumentUnitSearchParameter =
+  | "documentNumber"
+  | "fileNumber"
+  | "publicationStatus"
+  | "publicationDate"
+  | "scheduledOnly"
+  | "courtType"
+  | "courtLocation"
+  | "decisionDate"
+  | "decisionDateEnd"
+  | "withError"
+  | "myDocOfficeOnly"
+  | "withDuplicateWarning"
+  | "inboxStatus"
+</script>
+
+<template>
+  <div
+    v-ctrl-enter="handleSearchButtonClicked"
+    class="flex flex-col bg-blue-200"
+  >
+    <div
+      class="m-40 grid grid-flow-col grid-cols-[auto_1fr_auto_1fr] grid-rows-[auto_auto_auto] gap-x-12 gap-y-20 lg:gap-x-32"
+    >
+      <!-- Column 1 -->
+      <div class="ris-body1-regular ml-3 flex flex-row items-center">
+        Aktenzeichen
+      </div>
+      <div class="ris-body1-regular flex flex-row items-center">Gericht</div>
+      <div></div>
```

Comment:
nice-to-have: Wäre cool, wenn wir hier die empty `<div>`s loswerden würden. 

Man könnte mal mit `col-start-1` / `row-start-1` etc. experimentieren.

Noch cooler wäre es mit [`grid-template-area`](https://developer.mozilla.org/en-US/docs/Web/CSS/grid-template-areas) zu arbeiten, aber gibt keine docu von tailwind dafür. Es scheint aber möglich zu sein: https://github.com/tailwindlabs/tailwindcss/discussions/2784#discussioncomment-8428904

Reply:
Hab mal ein kleines Experiment gemacht, geht ganz gut:
```html
  <div
    class="grid grid-cols-[1fr_1fr_auto] grid-rows-[auto_auto] [grid-template-areas:'az-label_az-input_az-input''._._search-button']"
  >
    <div class="bg-red-400 [grid-area:az-label]">Aktenzeichen label</div>
    <div class="bg-blue-400 [grid-area:az-input]">Aktenzeichen Input</div>
    <div class="bg-yellow-400 [grid-area:search-button]">Searchbutton</div>
  </div>
```
<img width="603" alt="image" src="https://github.com/user-attachments/assets/339e6e1a-4c75-4d01-8c81-3c7f0402273b" />